### DISCUSSION TITLE
Optimization, bugs, frame gaps, lags

### DISCUSSION THREAD

**Discussion by Denrise** [FakeFishGames/Barotrauma#13913] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

After update 1.4.5.0 the game slows down and lags a lot, there were no such problems before. Regardless of game settings. At minimum settings the character moves smoothly, but the game slows down a lot. At maximum settings, severe frame tearing appears, the game turns into a slide show, and the character moves sharply. Vertical sync smooths things out a bit, but doesn't help. Also, with the new update, the camera in stories has become worse. Now I look at the game as an “observer” and the view has become huge, it’s uncomfortable to play. I see lights everywhere but the doors are closed. Before the update there were no such problems, I played with a friend on high settings.
Specifications of my Acer Ex2511G device:
Processor i3-5005U 2.00 GHz
Geforce 920M video card.
RAM 8 GB
![Снимок экрана (139)](https://github.com/FakeFishGames/Barotrauma/assets/168666852/9b53207d-3cb0-4403-94cc-8e189e337c8f)

### Reproduction steps

Constant problem.
1) At minimum light and particle settings, the character moves smoothly. But the game is slow.
2) At maximum light and particle settings there is a significant drop in performance. The character moves abruptly, the game turns into a slideshow.

Campaign problem:
Camera bug. The camera is in observer mode. However, I can interact with the character. It is very uncomfortable. And due to the large number of light sources, the game slows down.
The game has become unplayable on my device.
![Снимок экрана (139)](https://github.com/FakeFishGames/Barotrauma/assets/168666852/06281cc7-87d8-4cff-b8a7-a0c2e97c604e)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by Xardion495** [comment]
Confirm. I have the same. Impossible to play multiplayer

**Comment by BlackMoons** [comment]
To help diagnose performance issues, press F3 and type 'showperf' in the console. Post screenshots with the additional info it puts on your screen.

**Reply by Denrise** [reply]
If it helps. That's it....

![Uploading Снимок экрана (147).png…]()
![Uploading Снимок экрана (148).png…]()
![Uploading Снимок экрана (149).png…]()

**Reply by BlackMoons** [reply]
I think you need to wait for the files to finish uploading before hitting submit. Those links just lead back to this page.

**Reply by Denrise** [reply]
![Снимок экрана (147)](https://github.com/FakeFishGames/Barotrauma/assets/168666852/b0419ab7-8ce8-4c43-a6a8-fcb2dbe22cc7)
![Снимок экрана (148)](https://github.com/FakeFishGames/Barotrauma/assets/168666852/550f67c7-1650-48df-a630-993917f977b0)
![Снимок экрана (149)](https://github.com/FakeFishGames/Barotrauma/assets/168666852/02dfa2f2-0df1-4dca-bf58-b60f4d481a60)

**Comment by ghost** [comment]
> Processor i3-5005U 2.00 GHz

Worth noting this does not meet the minimum listed requirement of 2.4ghz on steam.
![image](https://github.com/FakeFishGames/Barotrauma/assets/86113712/3b179e13-94d4-4f21-bfd7-d61f9706b47b)

**Comment by Xardion495** [comment]
Works normal since the last update

**Reply by Denrise** [reply]
the camera was fixed. optimization problems remain. It got worse.

**Comment by Denrise** [comment]
I even did it like this. It doesn't help, the game just slowed down and continues. No matter how I change the settings. Nothing changes.
![Снимок экрана (186)](https://github.com/FakeFishGames/Barotrauma/assets/168666852/902eb48c-aeba-4312-b8c1-69d58754a0ef)
